dubbo-common/src/main/java/org/apache/dubbo/common/utils/StringUtils.java

```diff
@@ -343,6 +343,20 @@ public static boolean isBlank(String str) {
         return isEmpty(str);
     }
 
+    public static boolean isRealBlank(CharSequence cs) {

```

Comment:
can be replace is 

```
public static boolean isRealBlank(CharSequence cs) {
        int strLen;
        if (cs != null && (strLen = cs.length()) != 0) {
            for (int i = 0; i < strLen; ++i) {
                if (!Character.isWhitespace(cs.charAt(i))) {
                    return false;
                }
            }
        }
        return true;
    }
```

Reply:
Hi, what can be replaced by what?

Reply:
existing **isRealBlank** can be change with given one. It is not exact replacement, I have paste the whole code of isRealBlank with modified one so that you don't have to spend time modifying line by line. In the modified one I have remove the extra **return** statement because the **return** at the end of method is good enough for the purpose.